network/datadog_checks/network/network.py

```diff
@@ -433,6 +433,29 @@ def _check_linux(self, instance):
                     self._submit_netmetric(nstat_metrics_names[k][met], self._parse_value(netstat_data[k][met]),
                                            tags=custom_tags)
 
+        proc_conntrack_path = "{}/net/nf_conntrack".format(proc_location)
+        try:
+            with open(proc_conntrack_path, 'r') as conntrack_file:
+                # Starting at 0 as the last line has a line return
+                conntrack_count = 0
+                while 1:
+                    conntrack_buffer = conntrack_file.read(65536)
+                    if not conntrack_buffer:
+                        break
+                    conntrack_count += conntrack_buffer.count('\n')
+                self.gauge('system.net.conntrack.count', conntrack_count, tags=custom_tags)
+        except IOError:
+            self.log.debug("Unable to read %s. Skipping conntrack metrics pull.", proc_conntrack_path)
```

Comment:
Hi, should we change `conntrack` to `nf_conntrack` like formatted below?

Reply:
The reason I didn't put `nf_conntrack` is that one of the enhancements could be to also have separate count for ipv4 and ipv6 which would result in metrics looking like:
* `system.net.conntrack.count` => totall conntrack count
* `system.net.conntrack.ipv4.count` => total conntrack count for ipv4 connections only
* `system.net.conntrack.ipv6.count` => total conntrack count for ipv6 connections only

But if you prefer I can use nf_conntrack, just let me know if an enhancement like that would make sense or if we should rather put the ipv4 conntrack in `system.net.ipv4.conntrack.count` in which case `system.net.nf_conntrack.count` would probably make more sense. I have no preference at this point.

Reply:
Thanks for the details explanation 😄. I like your reasoning. Is adding the `ipv4`/`ipv6` logic something you want to do with this PR? I love the idea to be honest

Reply:
No I'd rather keep the 1st one as basic as possible for now. The split ipv4/ipv6 might be a bit trickier to do because depending on the systems they might not be in the same files.
So I'd rather do a separate PR for that.